GenerateToolingFeed/Helper.cs

```diff
@@ -194,5 +170,28 @@ public static string GetReleaseVersionFromTag(JObject json, string tag)
             4 => ["v4", "v0"],
             _ => throw new ArgumentNullException($"{majorVersion} is not a supported version.", nameof(majorVersion))
         };
+
+        public static string GetCliVersion(string path)

```

Comment:
do we still need this logic if we are just reading from metadata.json file?

Reply:
This is for feed v3 in case we need to generate it.